addons/mail/static/tests/qunit_suite_tests/models/file_uploader_tests.js

```diff
@@ -50,6 +53,70 @@ QUnit.test("no conflicts between file uploaders", async function () {
     await contains(".o_Chatter .o_AttachmentCard");
     await contains(".o_ChatWindow .o_Message .o_AttachmentCard");
 });
+
+QUnit.test('Arrows on nonviewable attachment', async function (assert) {
```

Comment:
Would rename to `"Chatter main attachment: can change from non-viewable to viewable"`.

This would help grasp the purpose of test just by reading the test name.